CardDictionaries/Rosetta/ROSShared.php

```diff
@@ -162,6 +162,21 @@ function ROSPlayAbility($cardID, $from, $resourcesPaid, $target = "-", $addition
     case "ROS248":
       AddCurrentTurnEffect($cardID, $currentPlayer);
       return "";
+    case "ROS207": case "ROS208": case "ROS209":
+    case "ROS176": case "ROS177": case "ROS178":
+    case "ROS189": case "ROS190": case "ROS191":
+    case "ROS198": case "ROS199": case "ROS200":
+    case "ROS201": case "ROS202": case "ROS203":
+      DealArcane(ArcaneDamage($cardID), 0, "PLAYCARD", $cardID, resolvedTarget: $target);
+      return "";
+    case "ROS055":
+    case "ROS056":
+    case "ROS057":
+      if (GetResolvedAbilityType($cardID, "HAND") == "I") {
+        AddCurrentTurnEffect($cardID . "," . 1, $currentPlayer, "ABILITY");
+        return "Arcane Twinning is amping 1";
+      }
```

Comment:
This is not ROS055. C&P error me thinks.

Reply:
Thanks for the catch. For whatever reason even when I give it the right IDs arcane twinnings instant ability doesn't work, so I think I'll just delete that whole block for now.